### ISSUE TITLE
MacType_2017_0614_0.exe flagged by Symantec, cannot install

### ISSUE THREAD

**Issue by jmbattle0** [snowie2000/mactype#298]
Good morning,

I attempted to update my office PC to the latest version of MacType, however the installer is flagged by Symantec Endpoint Protection, and will not proceed.

![mactype_symantec](https://user-images.githubusercontent.com/29031955/27265407-130c191e-54d0-11e7-90f0-623d8feead38.png)
I tried to manually extract the files from the AdvancedInstaller EXE using the /extract [path] switch, but Symantec intercepts everything before any files can be created. Unfortunately, there is no possibility of disabling Symantec in order to install the update.

I've reinstalled the 2016_0904 released (which Symantec does not flag), but would like to request a zip archive containing the latest files that can be manually copied over the 2016 installation.

Thank you in advance!

**Comment by ErmanSayin** [comment]
Have you tried creating an exclusion for the executable?

**Comment by jmbattle0** [comment]
Unfortunately, I don't believe this will be possible, as the Symantec installation is locked-down pretty securely.
I am able to login to Windows as Administrator, but most of the Symantec options are grayed-out/disabled.

I wonder what changed in the 2017 installer?  The 2016 versions worked without any problems.

**Comment by snowie2000** [comment]
I think the reason the 2016 installer is trusted is that it is already widely spreaded, and Symantec had enough time and evidence to make sure that it's safe. But 2017 installer on the other hand is relatively new, and not digital signed, hence not trusted.

I knew that digital signing may fix this problem but as you may know, it is expensive (over $100/yr) and complicated to apply.

**Comment by jmbattle0** [comment]
Thank you for your reply.

Yes, that explanation makes sense.

I appreciate that digitally signing the installer is expensive, impractical, and may not even resolve the issue.  However, would it be possible to offer a zip archive of the update files that can be manually extracted over an existing installation?  Does the 2017 installer create any special registry keys?

Thank you for your help, keep up the great work with MacType.

**Comment by snowie2000** [comment]
The installer doesn't write very specially registry keys, but I don't quite sure if it will affect updater as it's provided by advanced installer. And for these who are using Windows with UAC enabled, they have to grant mactype folder with "everyone - fully control" permission manully so that switching profiles and update can be processed normally. Right now, installer deals with all of that.

**Comment by snowie2000** [comment]
Anyway, I'll provide a zip archive on my file station for these who need it.

Of course not now, since I'm planning to release another patch soon.

**Comment by jmbattle0** [comment]
Thank you very much for your support!

**Comment by jmbattle0** [comment]
Just downloaded 0628 version, but unable to install...as you might expect.

Thanking you in advance for uploading the zip version to github or file station. ;)

**Comment by snowie2000** [comment]
You can get zipped archive here:
http://www.mactype.net/station/?dir=Portable
(Due to the cdn cache, you may not able to see this folder directly from outside)

**Comment by jmbattle0** [comment]
Thank you very much for the URL.

I was able to download the file, however Symantec again complained about the download.  It automatically flagged a few files when trying to extract the ZIP.

So I tried to download the ZIP on my phone, then copy the file via USB.  This was successful.  I backed-up my customer .ini file, uninstalled the old MacType, then restarted.  Confirming that MacType was no longer installed (text was aliased), I made a new MacType folder in Program Files and copied in the extracted contents of the ZIP (downloaded on my phone).  Finally, I copied my .ini.  I restarted once again, then ran MacTray, which had automatically launched but was using the default settings.  I selected my tweaked .ini and everything appears to be working as before.

However, IE11 remains unchanged (as mentioned in a separate post).
